config/default/webhookcainjection_patch.yaml

```diff
@@ -0,0 +1,15 @@
+# This patch add annotation to admission webhook config and
+# the variables $(CERTIFICATE_NAMESPACE) and $(CERTIFICATE_NAME) will be substituted by kustomize.
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook-configuration
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```

Comment:
This whole [file seems duplicated](https://github.com/rabbitmq/messaging-topology-operator/blob/super-streams-only/config/default/base/webhookcainjection_patch.yaml). The mutating webhook bit makes me feel a bit uneasy. We could remove the file as a whole, or remove the mutating webhook bit at least?

Reply:
Yeh, looks that way. I think kubebuilder added it in when I created a new api with the CLI. It's not referenced by any kustomize file, so I'll remove this one.